internal/lightcone/destruction/underthebluesky/underthebluesky.go

```diff
@@ -0,0 +1,59 @@
+package underthebluesky
+
+import (
+	"github.com/simimpact/srsim/pkg/engine"
+	"github.com/simimpact/srsim/pkg/engine/equip/lightcone"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	Check key.Modifier = "under-the-blue-sky"
+	Buff  key.Modifier = "under-the-blue-sky-buff"
+)
+
+func init() {
+	lightcone.Register(key.UndertheBlueSky, lightcone.Config{
+		CreatePassive: Create,
+		Rarity:        4,
+		Path:          model.Path_DESTRUCTION,
+		Promotions:    promotions,
+	})
+
+	modifier.Register(Check, modifier.Config{
+		Listeners: modifier.Listeners{
+			OnTriggerDeath: onTriggerDeath,
+		},
+	})
+
+	modifier.Register(Buff, modifier.Config{
+		StatusType: model.StatusType_STATUS_BUFF,
+		Duration:   3,
+	})
```

Comment:
Needs `Replace` or `ReplaceBySource` stacking behavior. In the current state, if a 2nd enemy is killed while the buff is up, the buff **will not** be refreshed. `Replace` will have the new one take over, effectively resetting the duration